src/frameworks/stencil.json

```diff
@@ -16,6 +16,11 @@
     "command": "stencil build",
     "directory": "www"
   },
+  "logo": {
+    "default": "/logos/stencil/light.png",
```

Comment:
Are there SVG versions of this logo available?

Reply:
There are, I just forgot to update this reference, so good catch